pkg/db/canary.go

```diff
@@ -109,7 +109,10 @@ func UpdateChecksStatus(ids []string, status models.CheckHealthStatus) error {
 	}
 	return Gorm.Table("checks").
 		Where("id in (?)", ids).
-		Update("status", status).
+		Updates(map[string]any{
+			"status":     status,
+			"deleted_at": time.Now(),
```

Comment:
shouldnt this be server now ?

Reply:
time.Now() also has tz context, so it should be the same

Reply:
Not if there is a clock skew between db and server

Reply:
👍🏽